.upgrade.yml

```diff
@@ -1,5 +0,0 @@
-mappings:
-  ApiKeyMemberExtension: Sminnee\ApiKey\ApiKeyMemberExtension
-  ApiKeyRequestFilter: Sminnee\ApiKey\ApiKeyRequestFilter
-  GridFieldAddApiKeyButton: Sminnee\ApiKey\GridFieldAddApiKeyButton
-  MemberApiKey: Sminnee\ApiKey\MemberApiKey
```

Comment:
Can you update the requestfilter map but leave this file? It helps people to upgrade their sites